ceph.go

```diff
@@ -58,22 +58,24 @@ type osdDumpOut struct {
 }
 
 type osdTreeOutNode struct {
-	ID       int     `json:"id"`
-	Type     string  `json:"type"`
-	Name     string  `json:"name"`
-	Reweight float64 `json:"reweight"`
-	Children []int   `json:"children"`
+	ID       		int     `json:"id"`
+	DeviceClass string  `json:"device_class"`
```

Comment:
Spacing seems weird here and below - did you run `go fmt` against this?

Reply:
Run it now